docs/self-hosted/kubernetes-migration-guide.md

```diff
@@ -144,8 +144,9 @@ After each scale command, use `kubectl get pods` again to verify that the redis
 ```
 export PV_NAME=$(kubectl get pvc redis-data-redis-0 -ojsonpath='{.spec.volumeName}')
 kubectl scale statefulset -n default redis --replicas 0
-rm /var/openebs/local/$PV_NAME/appendonly.aof
-cp packagist_redis.rdb /var/openebs/local/$PV_NAME/dump.rdb
+sudo rm -f /var/openebs/local/$PV_NAME/appendonly.aof
+sudo rm -rf /var/openebs/local/$PV_NAME/appendonlydir
+sudo cp packagist_redis.rdb /var/openebs/local/$PV_NAME/dump.rdb
```

Comment:
If you need to `cp` this with sudo, can the Redis container still read the file or does this cause a permission issue?

Reply:
If we decide to remove sudo from the example, then would at least mention that they might require `sudo`

Reply:
All the existing files were owned by root. Also the Redis process is running as root, so cp with sudo should not cause any issues at all. I also didn't have any issues when testing this.

As the users might set up this differently, it might make more sense to take sudo out and mentioned that in a comment.

Reply:
Thanks!